src/index.js

```diff
@@ -0,0 +1,50 @@
+import CarModel from './model/CarModel.js';
+import InputView from './view/InputView.js';
+
+function printOneGame(nameList, cars) {
+  for (let i = 0; i < nameList.length; i++) {
+    const carOutput = '-'.repeat(cars[i].position);
+    console.log(`${nameList[i]} : ${carOutput}`);
+  }
+  console.log('');
+}
+
+// 입출력 예시
+export async function run() {
+  const cars = [];
+
+  const nameList = await InputView.getNameList();
+  const count = await InputView.getCount();
+
+  nameList.forEach(name => {
+    const car = new CarModel(name);
+    cars.push(car);
+  });
+
+  console.log('\n실행 결과');
+  for (let i = 0; i < count; i++) {
+    for (let j = 0; j < nameList.length; j++) {
+      const currentCar = cars[j];
+      currentCar.go();
+    }
+    printOneGame(nameList, cars);
+  }
+
+  let winnerPosition = 0;
+  cars.forEach(car => {
+    winnerPosition = Math.max(car.position, winnerPosition);
+  });
```

Comment:
다음처럼 간소화할 수. 있습니다. 선언과 정의에 필요한 연산을 한번에 하면 let 을 쓸 일이 줄어듭니다.

```suggestion
  const winnerPosition = cars.reduce((maxPosition, car) => 
    Math.max(car.position, maxPosition), 0);
```

Reply:
훨씬 깔끔한것같습니다...! 항상 사용하던것만 사용하는 문제가 있었는데, 이번기회에 알고있는 지식을 늘릴 필요성을 느낀것같습니다. 피드백 감사합니다!